src/NLog.StructuredLogging.Json.Tests/EndToEnd/ViaLayout/WithFailingLayout.cs

```diff
@@ -57,9 +59,10 @@ public void ShouldSurviveWhenLayoutFails()
         }
 
         [Test]
-        [Category("NotInNetCore")]
         public void ShouldLogFailureWhenLayoutFails()
         {
+            InternalLoggingOn();
```

Comment:
We're not going to merge this, it's just exploration. Now that we know what the issue is:
https://github.com/NLog/NLog/issues/2057#issuecomment-293625641

We can make a cleaner PR